templates/super_glue/wic/templates.yaml

```diff
@@ -0,0 +1,46 @@
+dataset: super_glue
+subset: wic
+templates:
+  14e73f39-a0d1-44c2-b9a4-4e48f9f1608e: !Template
+    id: 14e73f39-a0d1-44c2-b9a4-4e48f9f1608e
+    jinja: "Determine if the word '{{word}}' used in the same way in the two\nsentences\
+      \ below? Yes, No?\n{{sentence1}}\n{{sentence2}}\n||| \n{% if label == 0 %} \n\
+      No\n{% elif label == 1 %}\nYes\n{% endif %}"
+    name: question-context-with-label
+    reference: Generalized question-context format with label
+  3503ead5-4fa5-4f77-95dc-f0c2ed3eecdc: !Template
+    id: 3503ead5-4fa5-4f77-95dc-f0c2ed3eecdc
+    jinja: "Does the word `{{word}}` used in same context in the two\nsentences below?\n\
```

Comment:
- used in *the* context
- i don't think "context describes correctly the task we are asking: Does the word xxx has the same meaning in these two sentences?

Reply:
- *Is* the word

Reply:
To me, at first, it was also not so obvious. But, I specifically add this prompt because the task is named as "Word in Context" (WiC). Do you still suggest removing it? @VictorSanh

Reply:
I could be completely wrong. We should ask someone for who English is the first language.
@stephenbach what's actually the sensical way to say it?